README.md

```diff
@@ -61,4 +61,18 @@ And you'll probably get a warning about the app trying to boot on port `80`. You
 
 You'll need to update some paths and IPs in the code.
 <br/><br/>
-## Someone send me proof you got it all running.
\ No newline at end of file
+## Someone send me proof you got it all running.
+
+## Docker
+It is also possible to install the app into a docker container.
+Clone the reporsitory. Navigate to the path and build the docker image.
+ ```
+docker image build -t babysleepcoach . 
+```
+Run the container (adjust the ports, URLs and path).
+```
+docker run -d -p7080:80 -p7081:8000 -e REACT_APP_BACKEND_URL=URL_OF_THE_DOCKER_CONTAINER:7081 -e CAM_STREAM_URL=rtsp://user:password@URL_OF_CAMERA:554/stream1 -e DEBUG=False -e OWL=False -e HATCH_IP=127.0.0.1 -v /PATH_TO_SLEEP_LOGS_ON_HOST/sleep_logs.csv:/usr/app/babysleepcoach/sleep_logs.csv --name babysleepcoach babysleepcoach
```

Comment:
This is getting a little big, maybe it's worth continuing the use of the `.env` file and pass the file to docker instead of every arg on the command line? e.g. https://docs.docker.com/engine/reference/commandline/run/#env

Reply:
Good idea. I also added the option with the .env file and added an example